addons/api/addon/serializers/credential.js

```diff
@@ -22,27 +22,29 @@ export default class CredentialSerializer extends ApplicationSerializer {
         return this.serializeSSHPrivateKey(...arguments);
     }
   }
-
   serializeUsernamePassword() {
     const serialized = super.serialize(...arguments);
     // Remove non-username_password type attributes
     delete serialized['attributes']['private_key'];
```

Comment:
Is there a reason some fields use square bracket notation while some use dot notation?

Reply:
Originally the code that was added in this PR used bracket notation to match the existing code but then was changed through a suggestion. Would you like me to change the rest of the serializer to use dot notation as well?

Reply:
Yeah I think it'd be good to be consistent!